webClient/src/app/editor/code-editor/monaco/hiliters/rexx.ts

```diff
@@ -0,0 +1,184 @@
+export const REXX_HILITE = {
+  // Set defaultToken to 'invalid' to see what you do not tokenize yet/
+  //or 'default' otherwise
+  defaultToken: 'default',
+  ignoreCase: true,
+
+  "autoClosingPairs": [
+    ["{", "}"], ["[", "]"], ["(", ")"], ["\"", "\""], ["'", "'"]
+  ],
+
+  "surroundingPairs": [
+    ["{", "}"], ["[", "]"], ["(", ")"], ["\"", "\""], ["'", "'"]
+  ],
+
+  functions: [
+    'abbrev', 'abs', 'address', 'arg', 'bitand', 'bitor', 'bitxor', 'b2x', 'center',
+    'centre', 'compare', 'condition', 'copies', 'c2d', 'c2x', 'datatype', 'date',
+    'dbcs', 'delstr', 'delword', 'digits', 'd2c', 'd2x', 'errortext', 'externals',
+    'find', 'form', 'format', 'fuzz', 'getmsg', 'index', 'insert', 'justify',
+    'lastpos', 'left', 'length', 'linesize', 'listdsi', 'max', 'min', 'msg',
+    'mvsvar', 'outtrap', 'overlay', 'pos', 'prompt', 'queued', 'random', 'reverse',
+    'right', 'setlang', 'sign', 'sourceline', 'space', 'storage', 'strip', 'substr',
+    'subword', 'symbol', 'syscpus', 'sysdsn', 'sysvar', 'time', 'trace', 'translate',
```

Comment:
Same for `address`. It is keyword, not a function.

Reply:
I believe it's both a keyword and a function, for some reason https://www.ibm.com/support/knowledgecenter/SSLTBW_2.2.0/com.ibm.zos.v2r2.ikja300/trac.htm

Reply:
I agree it's both.